app/kubernetes/components/kube-services/kube-services-item/kube-services-item.controller.js

```diff
@@ -69,8 +81,6 @@ export default class KubeServicesItemViewController {
       this.addPort();
     }
 
-    this.KubernetesApplicationPublishingTypes = KubernetesApplicationPublishingTypes;
-
```

Comment:
This line is required for the HTML template to work properly
See  app/kubernetes/components/kube-services/kube-services-item/kube-services-item.html#L2
```
<div
  ng-if="$ctrl.isAdmin()"
  class="small text-warning"
  ng-show="$ctrl.serviceType === $ctrl.KubernetesApplicationPublishingTypes.LOAD_BALANCER && !$ctrl.loadbalancerEnabled"
>
```